change/@fluentui-react-badge-16df5a7e-d608-42e5-803b-981231c94c7a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

I think this should be of type `none` since it only affects docs and doesn't need to bump the package.

```suggestion
  "type": "none",
```